### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by makosblade**
just a little cleanup

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -251,11 +250,10 @@ dependencies {
      testCompile 'junit:junit:4.12'
```
Comment (UnsungHero97) [inline build.gradle]: looks like you're doing single quote to double quote. you missed some :)

Comment (makosblade) [inline build.gradle]: yeah I was just doing the few near my cursor. It's really a pretty un-necessary change at the moment but I hate having hardcoded version numbers

**Comment by geekbeast** [comment]
LGTM! We should test out bumping Jackson version sometime soon 👍

**Comment by makosblade** [comment]
+1 yeah we should. I'll probably do that tomorrow and go through and clean this up a bit

**Pull request merged**